other/upgrade.php

```diff
@@ -2368,7 +2368,7 @@ function cmdStep0()
 	$create = $smcFunc['db_create_table']('{db_prefix}priv_check', array(array('name' => 'id_test', 'type' => 'int', 'size' => 10, 'unsigned' => true, 'auto' => true)), array(array('columns' => array('id_test'), 'primary' => true)), array(), 'overwrite');
 
 	// ALTER
-	$alter = $smcFunc['db_add_column']('{db_prefix}priv_check', array('name' => 'txt', 'type' => 'tinytext', 'null' => false, 'default' => ''));
+	$alter = $smcFunc['db_add_column']('{db_prefix}priv_check', array('name' => 'txt', 'type' => 'varchar', 'size' => 255, 'null' => false, 'default' => ''));
```

Comment:
The equivalent on line 645 uses `'size' => 4`. It doesn't really make a difference, since we drop the table in the very next command, but for the sake of consistency, we may as well use the same size in both cases. So, could you please change this from `'size' => 255` to `'size' => 4`?